de.bmarwell.jdyninwx.app/pom.xml

```diff
@@ -0,0 +1,171 @@
+        <executions>
+          <execution>
+            <id>copy-runtime-dependencies</id>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <phase>prepare-package</phase>
+            <configuration>
+              <excludeScope>test</excludeScope>
+              <includeScope>runtime</includeScope>
+              <prependGroupId>true</prependGroupId>
+              <useSubDirectoryPerScope>true</useSubDirectoryPerScope>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.jreleaser</groupId>
+        <artifactId>jreleaser-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-assemble</id>
+            <phase>package</phase>
+            <goals>
+              <goal>assemble</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <jreleaser>
+            <project>
+              <name>inwx-updater</name>
+              <tags>
+                <tag>cli</tag>
+                <tag>inwx</tag>
+              </tags>
+              <maintainers>
+                <maintainer>bmarwell</maintainer>
+              </maintainers>
+              <stereotype>${jreleaser.stereotype}</stereotype>
+              <java>
+                <mainModule>jdyninwx.app</mainModule>
+                <mainClass>${project.java.mainClass}</mainClass>
+                <multiProject>true</multiProject>
+              </java>
+            </project>
+            <distributions>
+              <app>
+                <type>JAVA_BINARY</type>
+                <executable />
+                <tags>
+                  <tag>cli</tag>
+                  <tag>inwx</tag>
+                </tags>
+                <artifacts>
+                  <artifact>
+                    <path>${project.build.directory}/jreleaser/assemble/app/java-archive/inwx-updater-${project.version}.zip</path>
+                  </artifact>
+                </artifacts>
+              </app>
+            </distributions>
+            <assemble>
+               <active>ALWAYS</active>
+               <javaArchive>
+                <app>
+                  <active>ALWAYS</active>
+                  <exported>true</exported>
+                  <archiveName>inwx-updater-${project.version}</archiveName>
+                  <executable>
+                    <name>inwx-updater</name>
+                  </executable>
+                  <formats>
+                    <format>ZIP</format>
+                    <!-- wait for https://github.com/jreleaser/jreleaser/issues/1247 -->
+                    <!--format>TAR_GZ</format-->
+                  </formats>
+                  <mainJar>
+                    <path>${project.build.directory}/${project.artifactId}-${project.version}.jar</path>
+                  </mainJar>
+                  <java>
```

Comment:
This block is inherited from `<jreleaser><project>`. Given that both values are the same you can safely remove it.